ntfy/cli.py

```diff
@@ -47,13 +48,23 @@ def run_cmd(args):
 
 def send_notification(message, args, config):
     ret = 0
+    emojized = None
+    orig = message
 
     for backend in config['backends']:
         module = import_module('ntfy.backends.{}'.format(backend))
 
+        backend_config = config.get(backend, {})
+        backend_config.update(args.option)
+        if backend_config.get('disable_emoji', 'false').lower() \
```

Comment:
This one lets a specific back end disable emojis.  Need orig bc of loop
On Jan 20, 2016 7:52 AM, "Daniel Schep" <notifications@github.com> wrote:

> In ntfy/cli.py
> <https://github.com/dschep/ntfy/pull/10#discussion_r50249806>:
>
> >
> >      for backend in config['backends']:
> >          module = import_module('ntfy.backends.{}'.format(backend))
> >
> > +        backend_config = config.get(backend, {})
> > +        backend_config.update(args.option)
> > +        if backend_config.get('disable_emoji', 'false').lower() \
>
> Do if not and remove the message = orig clause
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/dschep/ntfy/pull/10/files#r50249806>.
>